pkgs/servers/adguardhome/default.nix

```
@@ -1,13 +1,10 @@
-{ lib, stdenv, fetchurl }:
+{ lib, stdenv, fetchurl, system ? stdenv.targetPlatform }:
 
 stdenv.mkDerivation rec {
-  name = "adguardhome";
-  version = "0.105.2";
+  pname = "adguardhome";
+  version = "v0.106.3";
```

```suggestion
  version = "0.106.3";
```

version numbers in nix shouldn't start with a v.

---

Oh I missed that when committing, sorry